### DISCUSSION TITLE
Links in pages

### DISCUSSION THREAD

**Discussion by durandguru** [jpanther/lynx#7] [Q&A]
Hi, is it possible to use subpages with links? I can link to a subpage but I have not found out to place links on that sub-page. Thanks

**Comment by jpanther** [comment]
Hi @durandguru, this isn't something that is currently possible but I can look at adding it in a future update. The exampleSite has a `link` shortcode that I use to demo all the link styles. You could replicate something like that as a workaround but it's still a bit of a hack to get the styles to display cleanly.

**Comment by 0lIvIa09** [comment]
Is this a feature yet? I have tried to use the shortcode as a workaround but can't get it to work

**Comment by matheus3301** [comment]
I opened PR #195 to add this capability: https://github.com/jpanther/lynx/pull/195

It allows each `content/<name>/_index.md` branch bundle to render as an independent Lynx link profile at `/<name>/`, using the same components as the homepage. The implementation remains fully static and preserves existing `params.author` homepage configurations. Feedback is welcome.
